subscriptions/tasks.py

```diff
@@ -269,7 +269,7 @@ def send_message(context):
         "to_identity": subscription.identity,
         "delivered": "false",
         "resend": "true" if "resend_id" in context else "false",
-        "metadata": {},
+        "metadata": message.metadata,
```

Comment:
I think we should add/update a test here, to ensure that this works the way we expect it to, eg. modify a test so that the message metadata has some stuff in it, and then make sure that that same stuff is sent to the message sender.

Reply:
noted thanks